[2022-03-23 05:54:58,772] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: postgres_dag.get_data_uba manual__2022-03-23T10:54:38.586588+00:00 [queued]>
[2022-03-23 05:54:58,780] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: postgres_dag.get_data_uba manual__2022-03-23T10:54:38.586588+00:00 [queued]>
[2022-03-23 05:54:58,780] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-03-23 05:54:58,781] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-03-23 05:54:58,781] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-03-23 05:54:58,961] {taskinstance.py:1264} INFO - Executing <Task(_PythonDecoratedOperator): get_data_uba> on 2022-03-23 10:54:38.586588+00:00
[2022-03-23 05:54:58,963] {standard_task_runner.py:52} INFO - Started process 1182 to run task
[2022-03-23 05:54:58,968] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'postgres_dag', 'get_data_uba', 'manual__2022-03-23T10:54:38.586588+00:00', '--job-id', '22', '--raw', '--subdir', 'DAGS_FOLDER/postgres_conn.py', '--cfg-path', '/tmp/tmpr9ih4tyt', '--error-file', '/tmp/tmp_wqjglm8']
[2022-03-23 05:54:58,968] {standard_task_runner.py:77} INFO - Job 22: Subtask get_data_uba
[2022-03-23 05:54:59,189] {logging_mixin.py:109} INFO - Running <TaskInstance: postgres_dag.get_data_uba manual__2022-03-23T10:54:38.586588+00:00 [running]> on host LAPTOP-5HJG55H7.localdomain
[2022-03-23 05:55:00,628] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=postgres_dag
AIRFLOW_CTX_TASK_ID=get_data_uba
AIRFLOW_CTX_EXECUTION_DATE=2022-03-23T10:54:38.586588+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-03-23T10:54:38.586588+00:00
[2022-03-23 05:55:00,635] {base.py:70} INFO - Using connection to: id: airflow-universities. Host: training-main.cghe7e6sfljt.us-east-1.rds.amazonaws.com, Port: 5432, Schema: training, Login: alkymer, Password: ***, extra: {}
[2022-03-23 05:55:01,418] {python.py:175} INFO - Done. Returned value was: None
[2022-03-23 05:55:01,755] {taskinstance.py:1272} INFO - Marking task as SUCCESS. dag_id=postgres_dag, task_id=get_data_uba, execution_date=20220323T105438, start_date=20220323T105458, end_date=20220323T105501
[2022-03-23 05:55:01,962] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-23 05:55:01,978] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
